.github/workflows/build.yml

```diff
@@ -67,16 +67,14 @@ jobs:
           retention-days: 5
 
   test:
-    name: Test on ${{ matrix.operating-system }} with Composer ${{ matrix.composer }} and PHP ${{ matrix.php }}
+    name: Test with PHP ${{ matrix.php }}
     needs: fixture
-    runs-on: ${{ matrix.operating-system }}
+    runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       max-parallel: 10
       matrix:
-        operating-system: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
Right now the workflow seems to fail constantly. If reducing the number of OS's we test against means it will be less likely to fail and we will not longer ignore constant failing we should do it.